database/neo4j/neo4j.go

```diff
@@ -113,9 +118,30 @@ func (n *Neo4j) Run(migration io.Reader) (err error) {
 		}
 	}()
 
-	if _, err := neo4j.Collect(session.Run(string(body[:]), nil)); err != nil {
-		return err
+	if n.config.MultiStatement {
+		statements := bytes.Split(body, []byte{';'})
```

Comment:
Might as well make the separate an internal package level variable/constant

Reply:
Pulled into a constant